src/bindings/python/fluxacct/accounting/util.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2025 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import pwd
+
+
+def get_uid(username):
+    """
+    Get the userid for a given username. If the userid cannot be found, just return
+    the username.
+
+    Args:
+        username: The username.
+    """
+    try:
+        return pwd.getpwnam(username).pw_uid
+    except KeyError:
+        return str(username)
+
+
+def get_username(userid):
+    """
+    Get the username for a given userid. If the username cannot be found, just return
+    the userid as a string.
+    """
+    try:
+        return pwd.getpwuid(userid).pw_name
+    except KeyError:
+        return str(userid)
+
+
+def set_uid(username, uid):
+    """
+    Look up and set the userid for a given username. If the userid cannot be found for
+    this username, set it to 65534.
+
+    Args:
+        username: The username.
+        uid: The userid.
+    """
+    if uid == 65534:
+        fetched_uid = get_uid(username)
+
+        try:
+            if isinstance(fetched_uid, int):
+                uid = fetched_uid
+            else:
+                raise KeyError
+        except KeyError:
+            uid = 65534
+
+    return uid
```

Comment:
I don't understand this function and I don't think the docstring is accurate. It does nothing unless the passed uid is `65534`? Also it raises a `KeyError` just to catch it immediately?